src/db.rs

```diff
@@ -56,11 +56,9 @@ impl Deref for DieselPooledConn<'_> {
 }
 
 pub fn connect_now() -> ConnectionResult<PgConnection> {
-    let mut url = Url::parse(&crate::env("DATABASE_URL")).expect("Invalid database URL");
-    if dotenv::var("HEROKU").is_ok() && !url.query_pairs().any(|(k, _)| k == "sslmode") {
-        url.query_pairs_mut().append_pair("sslmode", "require");
-    }
-    PgConnection::establish(&url.to_string())
+    let url = crate::env("DATABASE_URL");
+    let url = connection_url(&url);
```

Comment:
```suggestion
    let url = connection_url(&crate::env("DATABASE_URL"));
```

There is no need to have separate variables for this.

Reply:
good point, should be fixed now :)